include/bcs/SakiyamaIonAdvectionBC.h

```diff
@@ -0,0 +1,46 @@
+//* This file is part of Zapdos, an open-source
+//* application for the simulation of plasmas
+//* https://github.com/shannon-lab/zapdos
+//*
+//* Zapdos is powered by the MOOSE Framework
+//* https://www.mooseframework.org
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef SakiyamaIonAdvectionBC_H
+#define SakiyamaIonAdvectionBC_H
+
+#include "IntegratedBC.h"
+
+class SakiyamaIonAdvectionBC;
+
+template <>
+InputParameters validParams<SakiyamaIonAdvectionBC>();
+
+class SakiyamaIonAdvectionBC : public IntegratedBC
+{
+public:
+  SakiyamaIonAdvectionBC(const InputParameters & parameters);
+
+protected:
+  virtual Real computeQpResidual();
+  virtual Real computeQpJacobian();
+  virtual Real computeQpOffDiagJacobian(unsigned int jvar);
+
+  Real _r_units;
+//  Real _r;
```

Comment:
Leftover comment, like in `SakiyamaElectronDiffusionBC.h`

Reply:
Removed.